### ISSUE TITLE
Smooth scrolling in Windows File Explorer

### ISSUE THREAD

**Issue by MajkelG** [jcbastosportela/logioptspp#12]
Hi, 
I was wondering if it is possible to setup the smooth scrolling in Windows File Explorer. Even if your app is working and enables smooth scrolling on every other window, the file explorer still scrolls in steps. It only works smoothly when I use a touchpad, but not my Logitech mouse.
I'll be thankfull if you would check this.

**Comment by jcbastosportela** [comment]
I don't think it is an easy task. This application only tries to trick Logitech software into thinking that other applications are chrome.exe. In this case logitech does send high resolution scroll events. Why and how isn't explorer process them as such? That I don't know.
